src/c++/uds/src/uds/murmurhash3.c

```diff
@@ -8,7 +8,24 @@
 
 #include "murmurhash3.h"
 
+#ifndef VDO_UPSTREAM
+#include <linux/version.h>
+#undef VDO_USE_ALTERNATE
+#if defined(RHEL_RELEASE_CODE) && defined(RHEL_MINOR) && (RHEL_MINOR < 50)
+#if (RHEL_RELEASE_CODE > RHEL_RELEASE_VERSION(10, 0))
+#define VDO_USE_ALTERNATE
+#endif
+#else /* !RHEL_RELEASE_CODE */
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(6, 11, 0))
+#define VDO_USE_ALTERNATE
```

Comment:
I'm not sure if it matters, but in other places the USE_ALTERNATE type macros are always used for the older-than-current versions of kernel APIs, and the code branch with no ALTERNATE macro defined is what is used for the latest kernel sources. It looks like you're seting up VDO_USE_ALTERNATE to be used for the newest kernels.